library/core/src/slice/mod.rs

```diff
@@ -3293,8 +3293,9 @@ mod private_slice_index {
     on(T = "str", label = "string indices are ranges of `usize`",),
     on(
         all(any(T = "str", T = "&str", T = "std::string::String"), _Self = "{integer}"),
-        note = "you can use `.chars().nth()` or `.bytes().nth()`
-see chapter in The Book <https://doc.rust-lang.org/book/ch08-02-strings.html#indexing-into-strings>"
+        note = "you can use `.chars().nth()` or `.bytes().nth()`. \
+                Also please see this chapter in The Book \
```

Comment:
```suggestion
                For more information, see chapter 8 in The Book: \
```

Reply:
Hmm, it looks like the literal newline here was counted before. So now it shows up all on one very long line. I think the newline should be added back:
```suggestion
                \nFor more information, see chapter 8 in The Book: \
```